l2-contracts/contracts/bridge/L2NativeTokenVault.sol

```diff
@@ -188,6 +188,9 @@ contract L2NativeTokenVault is IL2NativeTokenVault, Ownable2StepUpgradeable {
     function l2TokenAddress(address _l1Token) public view override returns (address expectedToken) {
         bytes32 expectedAssetId = DataEncoding.encodeNTVAssetId(L1_CHAIN_ID, _l1Token);
         expectedToken = tokenAddress[expectedAssetId];
+        if (expectedToken == address(0)) {
```

Comment:
I wanted to remove this. The l2TokenAddress is backwards compatible on the L2SharedBridgeLegacy, but on the L2NTV I only want to return the deployed address, otherwise the user should call the calculateCreate2TokenAddress